src/pages/view/TitleBar/ActionsBar.js

```diff
@@ -70,29 +70,35 @@ const ViewActions = ({
         return <Redirect to={redirectUrl} />
     }
 
+    const onRecordError = () => {
+        warningAlert.show({
+            msg: i18n.t('The dashboard could not be cached. Please try again.'),
```

Comment:
For consistency, I think the message `The dashboard couldn't be made available offline. Try again.` is clearer, as _caching_ is not referred to anywhere else.

Reply:
@cooper-joe Should it also be a critical message rather than a warning?

Reply:
I updated the PR description with a screenshot showing the new text plus using critical

Reply:
Looks good 👍